app/models/project.rb

```diff
@@ -52,6 +53,10 @@ def archived?
     status == "archived"
   end
 
+  def locked?
+    locked_at.present?
+  end
```

Comment:
I can revert this change, but was really simple to add (I updated some code that was using `project.locekd_at.nil?` directly)